tests/test-kubernetes.sh

```diff
@@ -36,76 +31,44 @@ kubectl_config() {
 kubectl_deploy() {
     kubeclt_clean
 
-    echo "Creating password..."
-    echo 'password' > password.txt
-    tr -d '\n' <password.txt >.strippedpassword.txt && mv .strippedpassword.txt password.txt
-
-    echo "Creating local volumes..."
-    kubectl create -f local-volumes.yaml
-
-    echo "Creating secrets..."
-    kubectl create secret generic mysql-pass --from-file=password.txt
-
-    echo "Creating MySQL Deployment..."
-    kubectl create -f mysql-deployment.yaml
-
-    echo "Creating Wordpress Deployment..."
-    kubectl create -f wordpress-deployment.yaml
+    echo "Running scripts/quickstart.sh"
+    "$(dirname "$0")"/../scripts/quickstart.sh
 
     echo "Waiting for pods to be running..."
     i=0
     while [[ $(kubectl get pods -l app=wordpress | grep -c Running) -ne 2 ]]; do
         if [[ ! "$i" -lt 24 ]]; then
-            echo "Timeout waiting on pods to be ready. Test FAILED"
-            exit 1
+            echo "Timeout waiting on pods to be ready"
+            test_failed "$0"
         fi
         sleep 10
         echo "...$i * 10 seconds elapsed..."
         ((i++))
     done
-
     echo "All pods are running"
 }
 
 verify_deploy() {
-    # Check Wordpress is running.
     IPS=$(bx cs workers "$CLUSTER_NAME" | awk '{ print $2 }' | grep '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')
 
     for IP in $IPS; do
-        while true
-        do
-            code=$(curl -o /dev/null -sw "%{http_code}\n" http://"$IP":30180/wp-admin/install.php -o /dev/null)
-            if [ "$code" = "200" ]; then
-                echo "Wordpress is up."
-                break
-            fi
-            if [ "$TRIES" -eq 10 ]
-            then
-                echo "Failed finding Wordpress. Error code is $code"
-                exit 1
-            fi
-            TRIES=$((TRIES+1))
-            sleep 5s
-        done
+        if [[ "$(curl -sw '%{http_code}' http://$IP:8080/api/v1/namespaces/default/services/wordpress/proxy/wp-admin/install.php -o /dev/null)" = "200" ]]; then
```

Comment:
travis doesn't like this line:

^-- SC2086: Double quote to prevent globbing and word splitting.

Reply:
Thanks for pointing that out! I con't believe my linter didn't catch it locally...